Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -26,10 +26,12 @@ namespace Acts {
 /// The default configuration only takes the best matching measurement without a
 /// cut on the local chi2.
 struct MeasurementSelectorCuts {
+  /// bins in |eta| to specify variable selections
+  std::vector<double> etaBins;
   /// Maximum local chi2 contribution.
-  double chi2CutOff = std::numeric_limits<double>::max();
+  std::vector<double> chi2CutOff{std::numeric_limits<double>::max()};
   /// Maximum number of associated measurements on a single surface.
-  size_t numMeasurementsCutOff = 1;
+  std::vector<int> numMeasurementsCutOff{1};
```

Comment:
This was the simplest option so I could define the `--ckf-selection-nmax` option using the `VariableIntegers`. I don't thing there is `VariableUnsignedIntegers` type. With that choice, it was just a question where to do the conversion. I did it [here](https://github.com/acts-project/acts/pull/1064/files/a88294b7494069a2dac35467bce11b36739ce927#diff-a767add0929eb1306bbca2878f5ad6a9acfcce2ef465810001a6a37973e27292R152).
I could also move it somewhere else, though probably needing a vector conversion.

Reply:
I since this is disconnected from the CLI inputs, I guess I would lean towards using `unsigned int` here and converting the CLI arguments before they hit `MeasurementSelector`. I can't think of any use-case where we would want to have negative number of measurements 😆

Reply:
OK, I fixed that.
It will still give a crazy-big cut if you do specify a negative number, but that's now just in the examples no longer in the Core.
Thanks for the suggestion.